scripts/codeCoverage.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+LAUNCH_DIR=`pwd`
+
+#  uncoment to enabled source blacklist: exmple --> SOURCE_BLACKLIST=".*ProbeTransmogrifier.cpp.*"
+HEADER_WHITELIST=
```

Comment:
I think you need a comment to explain why one would want to add a file to the white list.